app/src/main/java/org/greenstand/android/TreeTracker/api/models/responses/ResponseTypes.kt

```diff
@@ -0,0 +1,33 @@
+package org.greenstand.android.TreeTracker.api.models.responses
+
+import com.google.gson.annotations.SerializedName
+
+data class UserTree(
```

Comment:
Can we lease have the word Reponse at the end of these data classes so ppl know they are from API?